src/internal/v1api/providers.go

```diff
@@ -46,14 +46,14 @@ func NewProviderGenerator(p provider.Provider, destination string, gpgKeyLocatio
 func (p ProviderGenerator) VersionListingPath() string {
 	namespacePath := strings.ToLower(p.Provider.Namespace)
 	providerNamePath := strings.ToLower(p.Provider.ProviderName)
-	return filepath.Join(p.Destination, "v1", "providers", namespacePath, providerNamePath, "versions")
+	return strings.ToLower(filepath.Join(p.Destination, "v1", "providers", namespacePath, providerNamePath, "versions"))
```

Comment:
This is not necessary dye to the above ToLower.  The module changes should probably follow a similar pattern.

Reply:
ahh good point, I was lazy and didn't read it all :D let me do it this way